src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenNullCoalescingAssignmentTests.cs

```diff
@@ -1158,13 +1481,12 @@ public static void Main()
         C c5 = null;
         UseParam(c5 ??= default);
 
-        // Implicit converstion from default literal to nullable value type.
-        // This should issue a warning, as we're converting to A, not A0
         int? i2 = null;
-        UseParam(i2 ??= default); // warn
+        UseParamAsInt(i2 ??= default);
```

Comment:
>default [](start = 29, length = 7)

Please verify semantic model on `default`. Also, let's verify the resulting value in execution (could be done in a standalone test).
If I understood correctly, this is now `default(int)` (as opposed to `default(int?)`/`null` previously).